pkgs/development/ocaml-modules/morbig/default.nix

```diff
@@ -1,4 +1,8 @@
-{ lib, buildDunePackage, fetchFromGitHub, ocamlPackages }:
+{ lib, buildDunePackage, fetchFromGitHub
+
+## OCaml packages
```

Comment:
This comment is not accurate: `buildDunePackage` is also an “OCaml package”.

Reply:
Good point; fixed it.